### PR TITLE
Add visits data and queries

### PR DESCRIPTION
**Pull request by SergSkol**
- Add vets table, visits table
- Refactor relations with 'join tables'
- Prepare complex queries that answer analytical questions.

Screenshots:

image

image

image

image

image

image

image

image

image

### CODE REVIEW

File: data.sql [hunk 1]
```diff
@@ -38,3 +42,36 @@ update animals set owner_id=4 where name = 'Squirtle';
...
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Stephanie Mendez', 64, 'May 04, 1981');
+INSERT INTO vets (name, age, date_of_graduation) VALUES ('Jack Harkness', 38, 'Jun 08, 2008');
+
+INSERT INTO specializations (species_id, vets_id) VALUES (1, 1);
+INSERT INTO specializations (species_id, vets_id) VALUES (1, 3);
+INSERT INTO specializations (species_id, vets_id) VALUES (2, 3);
+INSERT INTO specializations (species_id, vets_id) VALUES (2, 4);
+
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 1, 'May 24, 2020');
+INSERT INTO visits (animals_id, vets_id, date_of_visit) VALUES (3, 3, 'Jun 22, 2020');
```
Comment (sja-thedude) [inline data.sql]: - [ ] Kindly make sure to write the correct date, it's July, not June for the month as per the requirements.

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @SergSkol, This is @sja-thedude your code reviewer.

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

768070f3-bc9a-4aba-b8bc-25793e00ea1e_text

## Highlights
✔️ Professional Readme
✔️ Correct gitflow

## Required 
- [ ] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: data.sql (hunk 1 above)
Comment (SergSkol) [reply data.sql]: Thank you, I've missed there. Now it's fixed. No changes in queries results.

**Review by mckent05 — APPROVED** [review]
Hi @SergSkol, 

# STATUS: APPROVED ✔️ ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations and well done! 🎉 
approved


## Highlights
- All linter checks are passing ✔️ ✔️ 
- Your PR is professional 👏 👏 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**